.github/workflows/commit-validation.yml

```
@@ -0,0 +1,105 @@
+              echo "Types: feat, fix, docs, style, refactor, perf, test, build, ci, chore, revert"
+            fi
+          done
+
+  pr-validation:
+    name: PR Title & Description Check
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
+    
+    steps:
+      - name: Validate PR title
+        run: |
```

actionlint (1.7.9)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)